.circleci/config.yml

```diff
@@ -11,15 +11,12 @@ jobs:
           command: |
             curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.15.2
       - run:
-          name: Install dependencies from offline mirror
-          command: PATH=$HOME/.yarn/bin:$PATH yarn install --offline
-      - run:
-          name: Run Continuous Integration checks
-          command: PATH=$HOME/.yarn/bin:$PATH yarn ci-check
+          name: Install dependencies
+          command: PATH=$HOME/.yarn/bin:$PATH yarn install
       - run:
           name: Confirm project can build
           command: |
-            PATH=$HOME/.yarn/bin:$PATH yarn build:external && yarn storybook:build
+            PATH=$HOME/.yarn/bin:$PATH yarn build
```

Comment:
Try adding `GATSBY_CPU_COUNT=2` (or 1) to hard-code the amount of CPUs available to the container.

CircleCI unfortunately reports more than are actually available (I think 8 or something) which then leads to some OOM issues.

Feel free to reach out on Twitter if I can help more; thanks!

Reply:
Thanks Dustin!